generate/generate.go

```diff
@@ -417,8 +417,12 @@ func (as *allServices) GeneralCode() ([]byte, error) {
 	pn("  if err := json.Unmarshal(b, &m); err != nil {")
 	pn("    return nil, err")
 	pn("  }")
-	pn("  for _, v := range m {")
-	pn("    return v, nil")
+	pn("  for k, v := range m {")
+	pn("    if k == \"count\" {")
+	pn("		continue")
+	pn("	} else {")
+	pn("		return v, nil")
+	pn("	}")
```

Comment:
Sure.. maybe my understanding of what you were trying to do here is a bit off. My understanding was that you are simply trying to return the content of the resources in the response - so for the listXXX, I assumed you should only get the list of resources returned (although I haven't checked this). 

I found that what I was getting back was - only - the count of templates and no state info of the template as we are only returning the first value in the map. Normally I think there should only be one or two keys in the top level of the JSON response so I put this check in in case the first key encountered is the `count`